src/libImaging/TiffDecode.c

```diff
@@ -405,6 +402,11 @@ ImagingLibTiffDecode(
     TIFF *tiff;
     uint16 photometric = 0;  // init to not PHOTOMETRIC_YCBCR
     int isYCbCr = 0;
+    UINT8 planarconfig = 0;
+    UINT8 planes = 1;
```

Comment:
Is there any reason to use `UINT8` for `planes`? It looks to me like it would make more sense to use `int`.

Reply:
Yeah, should be just `int` :facepalm: